bundled/boost-1.62.0/include/boost/random/binomial_distribution.hpp

```diff
@@ -278,18 +278,18 @@ class binomial_distribution {
         m = static_cast<IntType>((t+1)*p);
 
         if(use_inversion()) {
-            q_n = pow((1 - p), static_cast<RealType>(t));
+            _u.q_n = pow((1 - p), static_cast<RealType>(t));
```

Comment:
Hm. Are these changes they've also made upstream in BOOST?

Reply:
Ah, sorry. Should have read the PR text above.

Reply:
Yes, the PR I referenced is merged.